src/FantomasOptionsDialogPage.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace FantomasVs
+{
+    using Microsoft.VisualStudio.Shell;
+    using System.Windows;
+
+    public class FantomasOptionsDialogPage : UIElementDialogPage
```

Comment:
As far as I know VS extensions we still need this intermediate layer.

Reply:
Yes, we need it to host a custom UI in VS.